packages/ember-simple-auth/package.json

```diff
@@ -31,6 +31,7 @@
     "silent-error": "^1.0.0"
   },
   "devDependencies": {
+    "@ember/legacy-built-in-components": "^0.4.1",
```

Comment:
`@ember/legacy-built-in-components` shouldn't be a part of the `ember-simple-auth` addon.
It's an optional peerDependency of `ember-engines`.

Reply:
Devdependencies are not "a part of" on addon, they are everything needed to build and test an addon. Only dependencies are "part of" an addon, as in when you install an addon, you only get the dependencies.

`@ember/legacy-built-in-components` was required to pass the default scenario. It's not optional if you are testing with ember 4+ because ember-engines uses it and we are testing ember engines in the tests.

Reply:
ESA addon doesn't use either `@ember/legacy-builtin-components` or the `ember-engines`, the test apps do.
I understand this isn't shipped to the users but it shouldn't be here at all since it's simply not used inside the addon itself.

Reply:
Ah ok, let me try that.